src/apiutils.js

```diff
@@ -19,6 +19,9 @@
  * @fileoverview Contains util functions for api endpoints.
```

Comment:
Please explain a little more

You can add more lines to describe the file in detail